docs/index.rst

```diff
@@ -14,8 +14,9 @@ in alpha. For more information about Takahē, see
    :maxdepth: 2
    :caption: Contents:
 
-   installation
    features
+   installation
+   upgrading
```

Comment:
I put features to come first because it make more sense to showcase takahe's features before deciding on installing it, and also to have upgrading coming after installation without pushing features down in the menu.